com.jetbrains.Rider.yaml

```diff
@@ -0,0 +1,55 @@
+
+app-id: com.jetbrains.Rider
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: rider-desktop
+separate-locales: false
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --socket=session-bus
```

Comment:
Why session bus and org.freedesktop.Flatpak?

Reply:
The session bus is because Rider is a development tool for .NET, so I think it's pertinent. To communicate with Unity development for instance. But I might misunderstand its purpose.

`org.freedesktop.Flatpak` is because other Flatpaks of Jetbrains software (like PhpStorm and IntelliJ IDEA Ultimate) has it, so I thought they used it for something.

Reply:
I guess the idea is to allow to spawn commands on the host. No idea how widely it is used but let's keep it for consistency.